yarn.lock

```diff
@@ -3477,7 +3477,7 @@
     xhr "^2.2.0"
     xtend "^4.0.1"
 
-"@portis/web3@2.0.0-beta.55", "@portis/web3@^2.0.0-beta.54":
+"@portis/web3@^2.0.0-beta.54":
```

Comment:
why is this getting pulled into yarn.lock. I would expect all portis stuff to be removed.

Reply:
looks like its being used by a upstream dependency, `3id-connect`